core/src/main/java/org/apache/carbondata/core/metadata/datatype/DataType.java

```diff
@@ -99,6 +99,8 @@ public static char convertType(DataType dataType) {
       return DATE_CHAR;
     } else if (dataType == DataTypes.BYTE_ARRAY) {
       return BYTE_ARRAY_CHAR;
+    } else if (dataType == DataTypes.BINARY) {
```

Comment:
you can keep `||` in above `if` condition, no need of new if condition

Reply:
ok, done